lizmap/modules/lizmap/classes/lizmapProject.class.php

```diff
@@ -359,13 +359,40 @@ protected function readProject($key, $rep)
         }
 
         $groupsWithShortName = $qgs_xml->xpath("//layer-tree-group/customproperties/property[@key='wmsShortName']/parent::*/parent::*");
-        if ($groupsWithShortName && count($groupsWithShortName) > 0) {
+        if ($groupsWithShortName && count($groupsWithShortName)) {
             foreach ($groupsWithShortName as $group) {
                 $name = (string) $group['name'];
                 $shortNameProperty = $group->xpath("customproperties/property[@key='wmsShortName']");
-                if ($shortNameProperty && count($shortNameProperty) > 0) {
+                if (!$shortNameProperty) {
```

Comment:
in the patch for lizmap 3.5, this line is ` if (!$shortNameProperty || count($shortNameProperty) == 0) {`. Is this difference expected?  I think we should add the count test, as `$shortNameProperty[0]` may trigger a warning.

Reply:
sorry, it is true that `!$shortNameProperty` is true when the array is empty. r+